src/Parse/ParseUser.php

```diff
@@ -338,10 +342,11 @@ public function linkWithTwitter(
         $screen_name,
         $consumer_key,
         $consumer_secret = null,
-        $auth_token,
+        $auth_token, //@codingStandardsIgnoreLine
```

Comment:
Same as the other twitter one, just bump it between the two rather than right after one of the args.